.github/workflows/api-build.yml

```diff
@@ -7,10 +7,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
 
       - name: Setup Node 18.x Environment
-        uses: actions/setup-node@v2
+        uses: actions/setup-node@v4
         with:
           node-version: "18.x"
```

Comment:
I know 18.x is still in support, but it's only security support, 20 is the one in active support. Could look to move to that version if it's supported? Having issues over on Azure and terraform with 20 not being fully supported yet, not sure how well AWS/github actions are doing

Reply:
I'd like to use 20 but I've left at 18 for now to match the project, as we still develop + run octopus with node 18 at the moment. It would be good to bump this up to v20 for the whole project soon; I would include the github actions as part of that